tests/helpers.py

```diff
@@ -88,7 +130,20 @@ def check_subgraph_escrow_account(sender, receiver, totalAmountThawing, balance)
         'query': graphql_query,
         'variables': vars
     }
-    resp = requests.post(subgraph_endpoint, json=request_data)
+    resp = requests.post(endpoint, json=request_data)
+    keep_trying = 10
+    while(keep_trying > 0):
+        time.sleep(3)
+        resp = requests.post(endpoint, json=request_data)
+        if resp.status_code == 200:
+            data = json.loads(resp.text)["data"]["escrowAccounts"]
+            if len(data) != 0:
+                break
+            else:
+                keep_trying -= 1
+        else:
+            raise Exception(f"Request failed with status code: {resp.status_code}")
```

Comment:
This retry structure appears more than once. Consider refactoring it into a function, or consider using a library that already facilitates retries on failure, such as https://github.com/litl/backoff